src/geometry.cpp

```diff
@@ -79,4 +81,32 @@ namespace geometry {
 	  return v;
 
 	}
+
+	std::vector<Cell> bresenham_conversion(const Cell &start, const Cell &end){
+
+		const auto pixels = bresenham_conversion(start.x, start.y, end.x, end.y);
+
+		std::vector<Cell> pixels_return;
+
+		for (const auto &pixel:pixels) {
+			Cell temp {pixel[0],pixel[1]};
```

Comment:
Could you either do `pixels_return.emplace_back({pixel[0],pixel[1]})` or `pixels_return.emplace_back(Cell{pixel[0],pixel[1]})`
since the temporary doesn't need to exceed this scope

Reply:
Done